xml/System.Runtime.InteropServices.JavaScript/JSObject.xml

```diff
@@ -97,9 +97,9 @@
       </Parameters>
       <Docs>
         <param name="propertyName">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Returns value of the property as <see cref="T:System.Byte" /> array if the property exists, otherwise <see langword="null" />.</summary>
```

Comment:
```suggestion
        <summary>Returns the value of the specified property as a <see cref="T:System.Byte" /> array if the property exists, otherwise <see langword="null" />.</summary>
```

Reply:
Please change throughout this file.